### PR TITLE
fix(crash_dump): Fix hocon schema related to crash dump

### PR DESCRIPTION
**Pull request by k32**

### CODE REVIEW

File: apps/emqx_conf/etc/emqx_conf.conf [hunk 1]
```diff
@@ -27,12 +27,27 @@ node {
...
 
-  ## Dir of crash dump file.
+  ## Location of crash dump file.
   ##
-  ## @doc node.crash_dump_dir
+  ## @doc node.crash_dump_file
   ## ValueType: Folder
   ## Default: "{{ platform_log_dir }}/"
-  crash_dump_dir = "{{ platform_log_dir }}/"
+  crash_dump_file = "{{ platform_log_dir }}/erl_crash.dump"
```
Comment (k32) [inline apps/emqx_conf/etc/emqx_conf.conf:35]: It should be a file rather a directory

File: apps/emqx_conf/etc/emqx_conf.conf [hunk 2]
```diff
@@ -27,12 +27,27 @@ node {
   ## Default: "{{ platform_data_dir }}/"
   data_dir = "{{ platform_data_dir }}/"
 
-  ## Dir of crash dump file.
+  ## Location of crash dump file.
   ##
-  ## @doc node.crash_dump_dir
+  ## @doc node.crash_dump_file
   ## ValueType: Folder
   ## Default: "{{ platform_log_dir }}/"
```
Comment (thalesmg) [inline apps/emqx_conf/etc/emqx_conf.conf:34]: `/erl_crash.dump`?

File: apps/emqx_conf/etc/emqx_conf.conf [hunk 3]
```diff
@@ -27,12 +27,27 @@ node {
...
-  crash_dump_dir = "{{ platform_log_dir }}/"
+  crash_dump_file = "{{ platform_log_dir }}/erl_crash.dump"
+
+  ## The number of seconds that the broker is allowed to spend writing
+  ## a crash dump
+  ##
+  ## @doc node.crash_dump_file
+  ## ValueType: Folder
+  ## Default: "{{ platform_log_dir }}/"
```
Comment (thalesmg) [inline apps/emqx_conf/etc/emqx_conf.conf:42]: This is using the previous value

File: apps/emqx_conf/etc/emqx_conf.conf [hunk 4]
```diff
@@ -27,12 +27,27 @@ node {
...
+  crash_dump_seconds = 30s
+
+  ## The maximum size of a crash dump file in bytes.
+  ##
+  ## @doc node.crash_dump_file
+  ## ValueType: Folder
+  ## Default: "{{ platform_log_dir }}/"
```
Comment (thalesmg) [inline apps/emqx_conf/etc/emqx_conf.conf:49]: Reminder to change this too

**Review by thalesmg — APPROVED** [review]

**Review by qzhuyan — APPROVED** [review]

File: apps/emqx_conf/etc/emqx_conf.conf [hunk 5]
```diff
@@ -27,12 +27,27 @@ node {
...
-  crash_dump_dir = "{{ platform_log_dir }}/"
+  ## @doc node.crash_dump_file
+  ## ValueType: File
+  ## Default: "{{ platform_log_dir }}/erl_crash.dump"
+  crash_dump_file = "{{ platform_log_dir }}/erl_crash.dump"
+
+  ## The number of seconds that the broker is allowed to spend writing
+  ## a crash dump
+  ##
+  ## @doc node.crash_dump_file
```
Comment (thalesmg) [inline apps/emqx_conf/etc/emqx_conf.conf:40]: `crash_dump_seconds`?

File: apps/emqx_conf/etc/emqx_conf.conf [hunk 6]
```diff
@@ -27,12 +27,27 @@ node {
...
+  ## ValueType: seconds
+  ## Default: 30s
+  crash_dump_seconds = 30s
+
+  ## The maximum size of a crash dump file in bytes.
+  ##
+  ## @doc node.crash_dump_file
```
Comment (thalesmg) [inline apps/emqx_conf/etc/emqx_conf.conf:47]: `crash_dump_bytes`?

**Review by thalesmg — DISMISSED** [review]

**Review by thalesmg — APPROVED** [review]

**Pull request merged**